localization/pose_initializer/launch/pose_initializer.launch.xml

```diff
@@ -17,8 +18,10 @@
     <remap from="pose_reset" to="/initialpose3d"/>
     <remap from="ekf_trigger_node" to="/localization/pose_twist_fusion_filter/trigger_node"/>
     <remap from="ndt_trigger_node" to="/localization/pose_estimator/trigger_node"/>
-    <param name="map_loader_name" value="/map/pointcloud_map_loader"/>
+    <param name="map_height_fitter.map_loader_name" value="/map/pointcloud_map_loader"/>
```

Comment:
I believe there is a movement to extract parameters from launch files to a param.yaml file. Is it possible to revise these to do so?

Reply:
`/map/pointcloud_map_loader` is a specific topic name that is only used by autoware, so it feels strange to write it in param.yaml.
However, it is not good for pose_initializer to include other remaps and directly write the topic name in launch, so I would like to fix it together with other topic name remaps in another pull request. :+1:

Reply:
Got you ideas :+1: